models/Program.js

```diff
@@ -32,10 +32,14 @@ const ProgramSchema = new Schema({
     enum: ["6th", "7th", "8th", "9th", "10th", "11th", "12th", "College Freshman", "College Sophmore", "College Junior", "College Senior"],
     required: true
   },
-  program_id_number: {
+  program_id: {
     type: String,
     required: true
   },
+  student_list: {
```

Comment:
This should be a list of ObjectIDs instead of objects. I can't find a great example online but it should be something like [this](https://www.codegrepper.com/code-examples/javascript/type+mongoose.schema.types.objectid+ref+%27user%27).

```js
student_list: [{
  type: Schema.Types.ObjectId,
  ref: 'Student'
}],
```